libr/util/str.c

```diff
@@ -2476,11 +2476,11 @@ R_API char *r_str_between(const char *cmt, const char *prefix, const char *suffi
 	return NULL;
 }
 
-R_API bool r_str_startswith(const char *str, const char *needle) {
+R_API int r_str_startswith(const char *str, const char *needle) {
```

Comment:
Yeah, please check my comment on the PR about `if(r_str_startswith(..))` giving incorrect results.. how should I solve this?